README.md

```diff
@@ -160,4 +160,64 @@ $ docker ps
 you should see output similar to the following
 ```bash
 CONTAINER ID   IMAGE     COMMAND   CREATED   STATUS    PORTS     NAMES
-```
\ No newline at end of file
+```
+
+## Test
+
+Building the image
+```bash
+$ docker build --target test --tag my-test-image .
+```
+Run unit test in the "todo_app/test" directory
+```bash
+$ docker run my-test-image ./todo_app/test/test_view_model.py
+```
+You should see output similar to the following:
+============================= test session starts ==============================
+platform linux -- Python 3.8.8, pytest-6.2.1, py-1.10.0, pluggy-0.13.1
+rootdir: /app
+collected 6 items
+
+todo_app/test/test_view_model.py ......                                  [100%]
+
+============================== 6 passed in 0.68s ===============================
+
+Run integration test in the "todo_app/test" directory
+```bash
+$ docker run --env-file ./.env.test my-test-image todo_app/test/test_app.py
+```
+You should see output similar to the following:
+============================= test session starts ==============================
+platform linux -- Python 3.8.8, pytest-6.2.1, py-1.10.0, pluggy-0.13.1
+rootdir: /app
+collected 1 item
+
+todo_app/test/test_app.py .                                              [100%]
+
+============================== 1 passed in 0.58s ===============================
+
+Run E2E test in the "todo_app/test" directory
+```bash
+$ docker run --env-file ./.env my-test-image todo_app/test/test_trello_app.py
+```
+You should see output similar to the following:
+============================= test session starts ==============================
+platform linux -- Python 3.8.8, pytest-6.2.1, py-1.10.0, pluggy-0.13.1
+rootdir: /app
+collected 1 item
+
+todo_app/test/test_trello_app.py  * Serving Flask app "todo_app.app" (lazy loading)
+.                                       [100%]
+
+============================== 1 passed in 10.32s ==============================
+
+# Set up Travis CI 
+
+Please add following variables to your environment variables by visiting https://travis-ci.com/github/YOUR_USERNAME/YOUR_REPOSITORY_NAME/settings
+
+api_key
+token
+board_id
+todo_list_id
+doing_list_id
+done_list_id
```

Comment:
Keeping these values in the website's settings, but it's the kind of thing that's slightly nicer to include in the .travis.yml file instead (secrets can be encrypted via the command line tool)

Keeping as much config as possible in the .travis.yml file means:
- It's source controlled
- You know exactly what the config was for a given build. 
- You can have branches with different settings that don't affect any other branches' builds.

Reply:
As advised I have updated api_key and token with encrypted version.